addons/gamification/security/ir.model.access.csv

```diff
@@ -25,3 +25,9 @@ badge_user_employee,"Badge-user Employee",model_gamification_badge_user,base.gro
 badge_user_manager,"Badge-user Manager",model_gamification_badge_user,base.group_erp_manager,1,1,1,1
 badge_user_portal,"Badge-user Portal",gamification.model_gamification_badge_user,base.group_portal,1,1,1,0
 badge_user_public,"Badge-user Public",gamification.model_gamification_badge_user,base.group_public,1,0,0,0
+
+karma_rank_employee,"Karma Rank Employee",model_gamification_karma_rank,base.group_user,1,0,0,0
```

Comment:
Guidelines: gamification_karma_rank_access_user,gamification.karma.rank.access.user,model_gamification_karma_rank,base;group_user,1,0,0,0

Idea is: xml id is 'model'_access_'group' (user for base.group user is ok, otherwise employee if it conflicts with other groups), name is xml id with '.' instead of '_' . That way it is standard :) .